codalab/objects/metadata.py

```diff
@@ -56,6 +56,10 @@ def set_metadata_key(self, key, value):
         self._metadata_keys.add(key)
         setattr(self, key, value)
 
+    def remove_metadata_key(self, key):
+        if key in self._metadata_keys:
+            self._metadata_keys.remove(key)
```

Comment:
Don't you need to do `delattr` (mirroring what `set_metadata_key` does)?

Reply:
Good catch! Updated.